### ISSUE TITLE
Build for 32 bit platforms or only x64?

### ISSUE THREAD

**Issue by ralphtheninja** [Level/leveldown#554]
We need to modify so `prebuildify` also builds for 32 bit if this is the case.

**Comment by vweevers** [comment]
4.0.1:

![image](https://user-images.githubusercontent.com/3055345/50051608-36b12c80-0116-11e9-8243-ae54a58c66fb.png)

4.0.0:

![image](https://user-images.githubusercontent.com/3055345/50051620-6a8c5200-0116-11e9-800f-c99b383221b0.png)

3.0.2:

![image](https://user-images.githubusercontent.com/3055345/50051627-7d9f2200-0116-11e9-90f7-5a7aed7051f4.png)

**Comment by vweevers** [comment]
So for the latest release, 1.9% of downloads are 32 bit. IMO it's fine to say, those few people should compile binaries themselves.

**Comment by vweevers** [comment]
@ralphtheninja good to close (we have an action item in #553)?

**Comment by okdistribute** [comment]
Hi we're running into this issue because we are building for 32bit machines, we have some folks with old computers who live in remote locations. They have limited connectivity and likely will not be upgrading Windows to 64bit. Is it possible to do a prebuilt for electron? What do you suggest when packaging leveldown>5.0 with electron? electron-builder documentation is also quite sparse.. 

Here's the original issue reported from one of our partners. https://github.com/digidem/mapeo-desktop/issues/305#issuecomment-601513928

**Comment by okdistribute** [comment]
apparently building 32-bit builds of leveldown as a dependency to an electron app is challenging, https://github.com/digidem/mapeo-desktop/runs/521162496?check_suite_focus=true

**Comment by vweevers** [comment]
> Is it possible to do a prebuilt for win32?

Certainly. I'd be happy to help those folks and I don't see any technical challenge on this end. The only concern is increasing our npm package size, but (surprisingly) we haven't gotten a single compliant about that yet. Is it a concern for y'all, e.g. on slow networks?

**Comment by okdistribute** [comment]
Yeah, although now I'm trying your suggestion of running npm install on 32-bit and seeing if that works, even though the build machine is 64-bit

**Comment by okdistribute** [comment]
That didn't work for me so easily and I'm going to take a break from it now. How much would adding this increase the package size?
